### PR TITLE
Release v5.3.1-rc1

### PR DESCRIPTION
**Pull request by sellout**

### CODE REVIEW

**Review by nuttycom — APPROVED** [review]
utACK

**Review by r3ld3v — APPROVED** [review]
utACK 

Nice work!

**Review by str4d — APPROVED** [review]
utACK 761b764b516e5dff8a572bf2d2a7ee2d5844816d

File: doc/release-notes/release-notes-5.3.1-rc1.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+-----
+
+This release fixes an error "Assertion `uResultHeight == rewindHeight` failed" (#5958)
+that could sometimes happen when restarting a node.
+
+Memory Usage Improvement
+------------------------
+
+The memory usage of zcashd has been reduced by not keeping Equihash solutions for all
+block headers in memory.
```
Comment (daira) [inline doc/release-notes/release-notes-5.3.1-rc1.md:14]: We should add the `asOfHeight` RPC changes when we do the changelog for the v5.3.1 release.

**Review by daira — COMMENTED** [review]
Post-hoc ACK

**Pull request merged**